example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -389,7 +390,7 @@
 					"$(inherited)",
 					"$(PROJECT_DIR)/Flutter",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.airship.airshipExample;
+				PRODUCT_BUNDLE_IDENTIFIER = com.urbanairship.richpush;
```

Comment:
do we actually want this bundle ID in the wild?

Reply:
it matches our other samples

Reply:
https://github.com/urbanairship/ios-library/blob/2c70f023772d06ab87a8cfd80e67f0931bfcb300/SwiftSample/SwiftSample.xcodeproj/project.pbxproj#L1286